apps/prairielearn/src/sync/course-db.ts

```diff
@@ -876,6 +877,12 @@ async function loadInfoForDirectory<T extends { uuid: string }>({
         tolerateMissing: recursive,
       });
       if (info) {
+        if (info.data && 'clientFiles' in info.data) {
```

Comment:
Why is `clientFiles` checked here? How is that related to drafts?

Reply:
Removed in 562ccc7ca4238b061cc5fed79757979f8c0db35b.